lib/zoo_user_warden_strategy.rb

```diff
@@ -4,11 +4,10 @@ def valid?
   end
 
   def authenticate!
-    u = ZooniverseUser.authenticate(params['user']['display_name'], params['user']['password'])
-    if u.nil?
-      fail!("Could not log in")
-    else
-      success!(u.import)
-    end
+    imported_user = nil
```

Comment:
I'm pretty sure you don't need to set imported_user to nil here. In line 9 the assignment expression evaluates to nil when the if expression if false.